test/functionalTest/cases/1048_subscription_cache/three_subscriptions.test

```diff
@@ -268,7 +282,7 @@ Date: REGEX(.*)
 06. Statistics
 ==============
 HTTP/1.1 200 OK
-Content-Length: 488
+Content-Length: 51REGEX((3|5))
```

Comment:
Maybe the correct way of dealing with any message with variable unbounded payload from a logical pooint of view, i.e., the ones using at least REGEX() `.*`, `.+`, `\d+`, `\d*`, etc. is to use `Content-Length: REGEX(\d+)`.